devices/tuya.js

```diff
@@ -1677,7 +1677,11 @@ module.exports = [
             const endpoint = device.getEndpoint(1);
             // Enables reporting of physical state changes
             // https://github.com/Koenkk/zigbee2mqtt/issues/9057#issuecomment-1007742130
-            await endpoint.read('genBasic', ['manufacturerName', 'zclVersion', 'appVersion', 'modelId', 'powerSource', 0xfffe]);
+            try {
+                await endpoint.read('genBasic', ['manufacturerName', 'zclVersion', 'appVersion', 'modelId', 'powerSource', 0xfffe]);
```

Comment:
This is interesting, maybe TuYa changed their magic packet to something different. Do you have a TuYa gateway to check this with? (sniff traffic when the device pairs to the TuYa gateway: https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#with-cc2531

Reply:
Yes, I have the gateway. But only the CC2652P Sonoff dongle stick.

Reply:
Asked it here now: https://github.com/Koenkk/zigbee2mqtt/issues/12760#issuecomment-1181811294